src/main/play-doc/operation/ClusterStop.md

```diff
@@ -60,14 +60,31 @@ http://dcos-host/#services >> Services >> conductr >> More >> Suspend >> Suspend
 
 * If uninstalling, destroy the service:
 
+#### DC/OS 2.8, 2.9
```

Comment:
Perhaps this is a typo, but isn't this `1.8` and `1.9`?

Reply:
Oops! was thinking about the ConductR release